identity-service/src/app.js

```diff
@@ -134,20 +131,29 @@ class App {
 
       return { app: this.express, server }
     } else {
+      const audiusInstance = await this.configureAudiusInstance()
```

Comment:
nit - can we move this below the comment on line 136? that was meant to indicate this is a forked process